submissions/ik-web/friends-app/js/script.js

```diff
@@ -0,0 +1,132 @@
+document.addEventListener('DOMContentLoaded', () => {
+
+    const url = 'https://randomuser.me/api/?results=80&seed=fa21';
+    const usersBox = document.querySelector('.usersBox');
+
+    // ---------------------------------------------------------------
+    function createUserCard(name, photo, gender, age, phone, email) {
+        return `
+            <div class="userCard">
+                <div class="userCard__inner">
+                    <h2 class="userCard__userName">${name}</h2>
+                    <div class="userCard__userPhoto"><img src=${photo} alt="photo"></div>
+                    <div class="userCard__userInfo userInfo">
+                        <div class="userInfo__item">${gender}</div>
+                        <div class="userInfo__item">${age} years old</div>
+                        <div class="userInfo__item">
+                            <a href="tel:${phone}">${phone}</a>
+                        </div>
+                        <div class="userInfo__item">
+                            <a href="mailto:${email}">${email}</a>
+                        </div>                    
+                    </div>
+                </div>
+            </div>
+        `;
+    }
+    
+    function getUsersInfo() {
+        return fetch(url)
+        .then(response => response.json())
+        .then(response => response.results)
+        .catch(err => window.location.reload());
+    }
+
+    function addUserCardsOnPage(usersInfo) {
+        let userCards = '';
+        usersInfo.forEach(user => {
+            userCards += createUserCard(
+                `${user.name.first} ${user.name.last}`,
+                user.picture.large, 
+                user.gender, 
+                user.dob.age, 
+                user.phone,
+                user.email
```

Comment:
Too many arguments. Usually, function should have not more than 3 arguments

Reply:
I understood. I am sorry for this mistake ... ((
I changed the parameters of this function.